include/bitstream.h

```diff
@@ -81,6 +81,9 @@ size_t stream_flush(bitstream* stream);
 /* copy n bits from one bit stream to another */
 void stream_copy(bitstream* dst, bitstream* src, size_t n);
 
+/* return pointer to start of data array used by this bitstream */
+void * stream_begin(bitstream* s);
```

Comment:
Change the return type to `const void *`

Reply:
It is not different, I must have missed that function. I will remove this function and use stream_data.